website/pages/index.tsx

```diff
@@ -1,5 +1,327 @@
 import React from 'react';
+import { useRouter } from 'next/router';
+import { css, cx } from 'emotion';
+import facepaint from 'facepaint';
+import { Overline } from '@leafygreen-ui/typography';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useViewportSize } from '@leafygreen-ui/hooks';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+import { GridContainer, GridItem } from 'components/Grid';
+import News from 'components/News';
+
+const mq = facepaint(
+  Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`),
+  { literal: true },
+);
+
+const backdrop = css`
+  background-color: ${uiColors.gray.light3};
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+`;
+
+const layoutProperties = css`
+  ${mq({
+    width: ['calc(100% + 48px)', '100%', '100%', '1077px'],
```

Comment:
1077 is oddly specific. what is that derived from?

Reply:
Derived from @audroue